src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -492,6 +492,50 @@ export default class DefaultAudioVideoController
     );
   }
 
+  async replaceLocalVideo(): Promise<void> {
+    let videoStream: MediaStream | null = null;
+    try {
+      videoStream = await this.mediaStreamBroker.acquireVideoInputStream();
+    } catch (error) {
+      throw new Error('could not acquire video stream from mediaStreamBroker');
```

Comment:
Should we include the error message too?

Reply:
sounds good, will change.